resources/locales/en/translation.json

```diff
@@ -12,6 +12,8 @@
   "Parteifarbe": "party color",
   "Farbcode": "color code",
   "CSS Klassenname": "CSS class name",
+  "min. Stimmen in %": "min. votes in %",
+  "max. Stimmen in %": "max. votes in %",
   "Stimmen in %": "votes in %",
```

Comment:
There are some missing translations for other strings like "Hochrechnungsdaten" and "Stimmen in %"